library/src/main/java/com/getstream/sdk/chat/interfaces/MessageInputManager.java

```diff
@@ -0,0 +1,9 @@
+package com.getstream.sdk.chat.interfaces;
+import com.getstream.sdk.chat.rest.Message;
+
+public interface MessageInputManager {
```

Comment:
@adrian09h ideally a better name + a javadoc explaining what this interface is/does